### PR TITLE
chore: make unsuccessful toasts stickier

### LINKED ISSUE #207: Unsuccessful toasts should be sticky

### PR DESCRIPTION
**Pull request by rofe**
Fix #207

### CODE REVIEW

File: src/extension/app/store/bulk.js [hunk 1]
```diff
@@ -418,7 +418,7 @@ export class BulkStore {
           openUrlsLabel,
           copyUrlsCallback,
           copyUrlsLabel,
-          6000,
+          60000, // keep for 1 minute
```
Comment (rofe) [inline src/extension/app/store/bulk.js:421]: Though successful, we should show this toast for longer than a few seconds to give users time to click the copy or open URL buttons.

File: src/extension/app/utils/admin-client.js [hunk 2]
```diff
@@ -105,6 +105,24 @@ export class AdminClient {
...
+  /**
+   * Shows the error toast.
+   * @param {string} message The error message
+   * @param {string} variant The toast variant (positive, warning, negative)
+   */
+  showErrorToast(message, variant) {
+    this.appStore.showToast(
+      message,
+      variant,
+      undefined,
```
Comment (dylandepass) [inline src/extension/app/utils/admin-client.js:117]: Not something for this PR, but seems like a good reason to move to passing an object instead ;)

**Review by dylandepass — APPROVED** [review]

File: src/extension/app/utils/admin-client.js (hunk 2 above)
Comment (rofe) [reply src/extension/app/utils/admin-client.js:117]: I was thinking the same thing :)

**Pull request merged**